cli/functionary/template.yaml

```diff
@@ -0,0 +1,40 @@
+# (required): schema version for the package definition
+version: 1.0
+package:
+  # (required) human-friendly unique identifier; cannot be changed
+  name: "__PACKAGE_NAME__"
+
+  #   # (optional) defaults to name; determines how the package is displayed in the UI
+  #   display_name: string
+
+  #   # (optional) detailed description
+  #   description: string
+
+  # (required) language that the functions are written in
+  language: "__PACKAGE_LANGUAGE__"
+
+  functions:
+#    # similar to package level fields, but for the function
+#    - name: string
+#      display_name: string
+#      description: string
+
+#      # (required) Parameters that the function takes
+#      parameters:
+#          # (required) should match the name of the parameter in the actual function signature
+#        - name: string
+#          # (optional) defaults to name; determines how the parameter is displayed in the UI
+#          display_name: string
+#          # (optional) detailed description
+#          description: string
+#          # (required) data type of the parameter     <-- list of valid types TBD, so leave it at this for now
```

Comment:
The `<--` comments were my comments and not intended to make it into the final template :smile: .